src/foam/doc/ModelBrowser.js

```diff
@@ -186,11 +186,12 @@ foam.CLASS({
 foam.CLASS({
   package: 'foam.doc',
   name: 'PropertyInfo',
-  documentation: "Table view model to display model in printable model browser's",
+  documentation: 'Table view model to display' +
```

Comment:
I thought the original version was easier to read.

Reply:
Are we following eslint coding standards? Regardless I agree.